autode/conformers/conformers.py

```diff
@@ -234,6 +232,21 @@ def prune_diff_graph(
         logger.info(f"Pruned on connectivity {n_prev_confs} -> {len(self)}")
         return None
 
+    def remove_no_energy(self) -> None:
+        """Remove all conformers from this list that do not have an energy"""
+        n_conformers_before_remove = len(self)
+
+        for idx in reversed(range(len(self))):  # Enumerate backwards
+            if self[idx].energy is None:
+                del self[idx]
+
+        n_conformers = len(self)
+        if n_conformers == 0 and n_conformers != n_conformers_before_remove:
+            raise RuntimeError(
```

Comment:
Maybe change this to a custom specific exception from autodE exceptions so that if the user is happy with pruning removing everything, then try...except can be used to ignore that exception

Reply:
yeah, agree – cheers 👍🏼